frontend/src/pages/ConnectedPages/Game/UsersInGame/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { UserInGame } from "../../../../websockets/MessageType";
+
+import "./usersingame.css";
+
+interface IUsersInGame {
+  users: UserInGame[],
+}
+
+const maxUsersToDisplay = 14;
+const maxCharactersPerName = 8;
+
+const UsersInGame: React.FC<IUsersInGame> = ({ users: users }: IUsersInGame): JSX.Element => {
+  users.sort((p1, p2) => p2.correctAnswers - p1.correctAnswers)
+
+  const setRowClassName = (index: number): string => {
+    let className = "";
+    className +=
+      (index + 1) % 2 ? "usersInGameEvenRowColor" : "usersInGameParRowColor";
+
+    return className;
+  };
+
+  const emojiPrefix = (index: number): string => {
+    const ranking = index + 1;
+    if (ranking === 1) {
+      return "🥇"
+    }
+    if (ranking === users.length) {
+      return "😴"
+    }
+    if (ranking === 2) {
+      return "🥈"
+    }
+    if (ranking === 3) {
+      return "🥉"
+    }
+    return ""
```

Comment:
Might be a little too much but I'm suggesting just for the fun of it, feel free to ignore ^^
```suggestion
    return "💩"
```

Reply:
Yeah, I think it would be too much to have emojis in front of every usernames. I actually thought of using "💩" instead of "😴" for the player with the worst score but in the end I didn't like the idea of having a poop in the application, which might not be good taste for everyone. :see_no_evil: